src/libraries/token/VariableSupplyToken.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/token/StandardToken.sol';
+
+
+contract VariableSupplyToken is StandardToken {
+    using SafeMath for uint256;
+
+    event Mint(address indexed target, uint256 value);
+    event Burn(address indexed target, uint256 value);
+
+    function mint(address _target, uint256 _amount) internal returns (bool) {
```

Comment:
Okay, I guess I misunderstood what this is for.  You said a while back that when users call disputeBondToken.withdrawToBranch(), we should be minting coins in order to achieve 2x target payout for bond holders during a fork, but this is just intended for ReportingTokens?